src/Service/ProductHandler.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace App\Service;
+
+use App\Entity\ProductInterface;
+
+class ProductHandler
```

Comment:
This "helper" violates SOLID.
Also, please pay attention to the arguments' order and their names.
The first method applies `$products` as the first argument. The second method uses `array $productArray` as a second argument. This is inconsistent.